### DISCUSSION TITLE
How to disable page scrolling on one page

### DISCUSSION THREAD

**Discussion by jm6271** [lepoco/wpfui#1274] [Q&A]
I have a WPF-UI app with a NavigationView control with several pages. On one of the pages I want the content to fill the whole page from top to bottom without ever getting taller than the page, so the user doesn't have to scroll to see the bottom of the page. I only want this on one of the pages, not all of them. How do I disable the scrollbar on one of the pages?

**Comment by mani9876** [comment]
Hello!
I have the exact same question - how did you solve this issue?

Thanks!

**Comment by jm6271** [comment]
I didn't find a way to disable scrolling on a page, but here's the solution I did find.  I have a grid with two rows. The top row has a star height and the bottom row has an auto height. I have a card control in the top row with a listview on the card. The listview would grow as items were added and get taller than the page, creating the scrollbar. My fix was to bind the MaxHeight property of the listview to a property that was the height of the card minus 36 (about the amount of padding around the listview on the card). You can look at the code here: https://github.com/jm6271/MyMoney/blob/87b080a66fe0294f51c2c6937d28f56be3d0da74/MyMoney/Views/Pages/AccountsPage.xaml#L53

So the only solution I can figure out is to make sure that the content of any layout controls like stackpanels and grids that can grow dynamically need to be somehow limited so they can't grow off the page.
